ops.h

```diff
@@ -362,12 +362,30 @@ static HWY_NOINLINE HWY_MAYBE_UNUSED void RMSNorm(
 static HWY_NOINLINE HWY_MAYBE_UNUSED void RMSNorm(
     const float* HWY_RESTRICT x, const hwy::bfloat16_t* HWY_RESTRICT weight,
     float* HWY_RESTRICT out, size_t size) {
+  namespace hn = hwy::HWY_NAMESPACE;
+
   constexpr float eps = 1e-6f;
-  float ss = SquaredL2(x, size);
-  ss = 1.0f / sqrtf(ss / StaticCast<float>(size) + eps);
-  for (size_t j = 0; j < size; j++) {
-    // Note 1.0f centering here
-    out[j] = (1.0f + hwy::F32FromBF16(weight[j])) * (ss * x[j]);
+  constexpr size_t unroll_size = 2;
```

Comment:
For constexpr please use kUnrollSize naming :)

Reply:
I 've set up Clang-Tidy to enforce the kCamelCase style for constexpr variables in the codebase, aiming for automated style checking and corrections.
Maybe in future it can be applied to fix all constexpr styles